osu.Game/Overlays/Comments/CommentEditor.cs

```diff
@@ -167,84 +168,75 @@ private void load(OverlayColourProvider colourProvider)
             };
         }
 
-        private partial class CommitButton : LoadingButton
+        private sealed partial class CommitButton : RoundedButton
         {
             private const int duration = 200;
 
-            public readonly BindableBool IsBlocked = new BindableBool();
-
-            public override bool PropagatePositionalInputSubTree => !IsBlocked.Value && base.PropagatePositionalInputSubTree;
-
-            protected override IEnumerable<Drawable> EffectTargets => new[] { background };
+            private readonly LoadingSpinner spinner;
+            private OsuSpriteText text = null!;
 
-            private readonly string text;
-
-            [Resolved]
-            private OverlayColourProvider colourProvider { get; set; }
+            public readonly BindableBool IsBlocked = new BindableBool();
 
-            private OsuSpriteText drawableText;
-            private Box background;
-            private Box blockedBackground;
+            private bool isLoading;
 
-            public CommitButton(string text)
+            public bool IsLoading
```

Comment:
This is too close to the existing `Drawable.IsLoaded` terminology to the point it's confusing. Can you rely on that instead, and move the `set` content here into `LoadComplete`?

Reply:
This property controls loading spinner presence on submit button. Right now it's used only by tests, so restructured this a little. Does it look better now?

Reply:
I don't see how it's changed. I still disagree with it existing. If you're looking to delay load in a test, doing so in a mock API request may be the more correct way.